### PR TITLE
Typo

### PR DESCRIPTION
**Pull request by MikeOpenHWGroup**
Fixed a simple typo in the README.

Signed-off-by: MikeOpenHWGroup <mike@openhwgroup.org>

### CODE REVIEW

**Review by GTumbush — COMMENTED** [review]
Approved!

**Review by GTumbush — COMMENTED** [review]
Approved

**Review by GTumbush — APPROVED** [review]
Approved.

**Comment by MikeOpenHWGroup** [comment]
Hey @GTumbush, Eclipse now recognizes you as a Committer, so you should be able to formally approve (and merge) this PR.

**Comment by GTumbush** [comment]
Are any tests kicked off on the Metrics cloud as a result of PR's?  If so, how do I see the result of this PR's testing?

**Comment by MikeOpenHWGroup** [comment]
A "ci_check" regression will be launched when the pull-request is merged.   I see that you have reviewed and approved this pull-request so your next step is to merge it.

Currently, the only person who can see these regressions is me.  @aimeepsutton and I will hopefully get that fixed on Monday.

**Review by GTumbush — APPROVED** [review]
Approved

**Comment by GTumbush** [comment]
Running ci_check after merging seems a step too late.  master is already broken.  Steve and I talked about kicking off a ci_check as a result of a PR. 

I do not see a merge button. Still saying "Merging is blocked.

**Review by GTumbush — APPROVED** [review]
Approved

**Review by GTumbush — APPROVED** [review]
Approved

**Comment by MikeOpenHWGroup** [comment]
TaDa!   Thanks @GTumbush.  You have no idea how great it is to have that typo _finally_ fixed.  :wink: 

You are now officially an OpenHW Group Committer.  (We should get T-shirts made up.)

**Pull request merged**
